internal/pkg/api/handleAck.go

```diff
@@ -333,9 +333,9 @@ func (ack *AckT) handlePolicyChange(ctx context.Context, zlog zerolog.Logger, ag
 			Str("agent.policyId", agent.PolicyID).
 			Int64("agent.revisionIdx", currRev).
 			Int64("agent.coordinatorIdx", currCoord).
-			Str("rev.policyId", rev.PolicyID).
-			Int64("rev.revisionIdx", rev.RevisionIdx).
-			Int64("rev.coordinatorIdx", rev.CoordinatorIdx).
+			Str("rev_policyId", rev.PolicyID).
+			Int64("rev_revisionIdx", rev.RevisionIdx).
+			Int64("rev_coordinatorIdx", rev.CoordinatorIdx).
```

Comment:
I don't want to hold off the fix, but isn't the convention to use `.` to separate the names in the log fields? Shouldn't we keep it, then perhaps finding another name for these fields?